components/kyma-environment-broker/Dockerfile.job

```diff
@@ -20,6 +20,5 @@ RUN apk --update add ca-certificates
 FROM scratch
 LABEL source = git@github.com:kyma-project/control-plane.git
 
-ARG BIN
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-COPY --from=build /bin/${BIN} /bin/${BIN}
+COPY --from=build /bin/main /bin/main
```

Comment:
It will break all the jobs except environment cleanup job if you change imaga reference in the yaml.